lib/Lib.IntTypes.fst

```diff
@@ -131,6 +268,11 @@ let add_mod #t #l a b =
   | U32  -> UInt32.add_mod a b
   | U64  -> UInt64.add_mod a b
   | U128 -> UInt128.add_mod a b
+  | S8 -> Int8.int_to_t (Int.add_mod (Int8.v a) (Int8.v b))
```

Comment:
There are no `_mod` operators in `FStar.IntN` modules for good reason: there is no simple way of implementing them in C short of instructing the compiler to use wrapping semantics for signed integer under/overflow.
If such operators are generally useful, we could add them and make extraction fail unless passing some option to KreMLin (so as to not use wrapping semantics unless explicitly requested), with a visible warning that it will be unsound to compile the code without wrapping semantics.

For the time being, I think we should restrict `_mod` operators to `unsigned` types.